.github/scripts/update_docs.py

```
@@ -0,0 +1,217 @@
+# ---------------------------------------------------------------------------
+ 
+def update_file(client: anthropic.Anthropic, filepath: str) -> None:
+    print(f"  Reading {filepath}...")
+    try:
+        with open(filepath, "r", encoding="utf-8") as f:
+            original = f.read()
+    except FileNotFoundError:
+        print(f"  WARNING: {filepath} not found — skipping.")
+        return
+ 
+    print(f"  Sending to Claude...")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)